lighthouse-core/gather/gatherers/dobetterweb/response-compression.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /**
+  * @fileoverview Determines optimized gzip/br/deflate filesizes for all responses by
+  *   checking the content-encoding header.
+  */
+'use strict';
+
+const Gatherer = require('../gatherer');
+const zlib = require('zlib');
```

Comment:
How does the built-in browserify? Do we need to do anything special for extension/DevTools?